projects/plugins/jetpack/jetpack-user-content-tracking-redirects.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * User Content Tracking Redirects
+ *
+ * The purpose of this file is to track user generated link clicks on the emails and redirect them to the original URL.
+ * This is done by generating an iframe pointing to the track and redirect logic in .com.
+ *
+ * @package automattic/jetpack
+ */
+
+add_action( 'init', 'jetpack_user_content_tracking_redirect' );
+
+/**
+ * Generate the iframe to track and redirect the user generated link clicks.
+ */
+function jetpack_user_content_tracking_redirect() {
+	if ( ! isset( $_SERVER['QUERY_STRING'] ) ) {
```

Comment:
Of course, it needs to check for the existence of `action=redirect` or something similar.

Reply:
Sure! Done 👍